README.md

```diff
@@ -47,10 +47,56 @@ For its [Digital section](https://www.decathlon.design/726f8c765/p/6145b2-overvi
 
 ### SPM
 
+Add Vitamin to the dependencies array of your package:  
 ```swift
 dependencies: [
-    .package(url: "https://github.com/Decathlon/vitamin-ios.git", .exact("0.10.0"))
-]
+  .package(url: "https://github.com/Decathlon/vitamin-ios.git", .exact("0.10.0")),
+  // Any other dependencies...
+],
+```
+Then, add Vitamin to the dependencies array of any target that depends on Vitamin.  
+For SwiftUI:  
+```swift
+.target(name: "YourSwiftUITarget",
+        dependencies: [
+            .product(name: "VitaminSwiftUI", package: "vitamin-ios"),
+            // Any other dependencies...
+        ]),
+```
+For UIKit:  
+```swift
+.target(name: "YourUIKitTarget",
+        dependencies: [
+            .product(name: "Vitamin", package: "vitamin-ios"),
+            // Any other dependencies...
+        ]),
+```
+
+A full example:  
+```swift
+let package = Package(
+    name: "YourPackage",
+    products: [
+        // Your products
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Decathlon/vitamin-ios.git", .exact("0.10.0")),
```

Comment:
Take care about the replication of your latest version number to avoid any error in the future.

Reply:
We have a script that bumps automatically the version for our other dependency tool (Cocoapods), we could easily add a 'sed' to bump the version here too, that would allow us not to take care of it manually.

Reply:
A small update was needed due to a wrong regex 😇 It's now solved.